voltaire_bundler/cli_manager.py

```
@@ -776,49 +783,77 @@ def init_logging(args: Namespace):
-            "eth_getCode",
-            [bundler_address, "latest"],
-        )
-        if "result" not in bundler_code_res:
+        raw_secrets = [s.strip() for s in args.bundler_secret.split(",")]
+        if len(raw_secrets) == 1:
+            pk = raw_secrets[0]
+            addr = public_address_from_private_key(pk)
+            per_ep_secrets = {label: (addr, pk) for label in ENTRYPOINT_LABELS}
+        elif len(raw_secrets) == 4:
+            per_ep_secrets = {}
+            for label, pk in zip(ENTRYPOINT_LABELS, raw_secrets):
```

Ruff (0.15.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)